Infomaniak Public Cloud 01 (S3).cyberduckprofile

```diff
@@ -37,16 +37,8 @@
         <false/>
         <key>Anonymous Configurable</key>
         <false/>
-        <key>Region</key>
-        <string>auto</string>
-        <key>Regions</key>
-        <array>
-            <string>auto</string>
-        </array>

```

Comment:
It is fine to omit these if the default `Region` is set above.

Reply:
I've delete this part from all profiles